src/OrchardCore.Themes/TheAgencyTheme/ResourceManifest.cs

```diff
@@ -53,6 +53,11 @@ public void BuildManifests(IResourceManifestBuilder builder)
                 .SetCdnIntegrity("sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T", "sha384-t4IGnnWtvYimgcRMiXD2ZD04g28Is9vYsVaHo5LcWWJkoQGmMwGg+QS0mYlhbVv3")
                 .SetVersion("4.3.1");
 
+            manifest
+                .DefineStyle("bootstrap-oc")
+                .SetUrl("~/TheAgencyTheme/css/bootstrap-oc.min.css", "~/TheAgencyTheme/css/bootstrap-oc.css")
+                .SetVersion("4.3.1");
```

Comment:
Why this version? Why not just "1" ? If you wanted to follow BS, then what will we do if we actually need to change this file, and we don't change BS?

Reply:
OK, bootstrap-oc version 1